addons/rivet/devtools/dock/playtest_tab.gd

```diff
@@ -65,4 +65,23 @@ func _update_warnings() -> void:
 		else:
 			warning.visible = true
 			error.visible = false
-		return
\ No newline at end of file
+			_generate_public_auth_token(current_namespace)
+		return
+		
+func _generate_dev_auth_token(ns) -> void:
+	for i in connection_type.tab_count:
+		connection_type.set_tab_disabled(i, true)
+	namespace_selector.disabled = true
+
+	var result = await RivetPluginBridge.get_plugin().cli.run_command(["sidekick", "get-namespace-dev-token", "--namespace", ns.name_id])
+	if result.exit_code != 0 or !("Ok" in result.output):
+		print("Error: " + result.output)
```

Comment:
(cc @AngelOnFira ) is there a plan to make this type of error handling standard in the `run_command` fn?